app/controllers/channels/fetch_controller.rb

```diff
@@ -0,0 +1,12 @@
+class Channels::FetchController < ApplicationController
+  before_action :login_required
+
+  def create
+    channel = Channel.find(params[:channel_id])
+    Channel.add(channel.feed_url)
+    ChannelItemsUpdaterJob.perform_later(channel_id: channel.id)
```

Comment:
```suggestion
    ChannelItemsUpdaterJob.perform_later(channel_id: channel.id, mode: :all)
```

にするとフィード内の全 Item をスキャンする処理にはできるのだけれど、ポッドキャストだと過去の 100 をこえるエピソードを全件チェックすることになってまあまあ重い。ので、差分チェックな処理にしてあります。

全件チェックしたい！というユースケースが発見されたら、そのときに考えるッピ！